pycqed/measurement/detector_functions.py

```diff
@@ -421,6 +421,8 @@ def __init__(self, UHFQC=None, detectors=None):
         self.UHFs = [d.UHFQC for d in self.detectors]
         self.UHF_map = {UHF.name: i
                    for UHF, i in zip(self.UHFs, range(len(self.detectors)))}
+        # to be used by MC.get_percdone()
+        self.acq_data_len_scaling = 1
```

Comment:
As this is used by the MC, it is expected to be there for all detector functions. It should therefore be in the base detector function class.

Reply:
Fixed in latest commit.